config/database.yml

```diff
@@ -64,10 +64,10 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  url: <%= ENV['DATABASE_URL_TEST'] || ENV['DATABASE_URL'] || ''%>
-  database: PracticalDeveloper_test<%= ENV['TEST_ENV_NUMBER'] %>
+  url: <%= ENV.fetch("DATABASE_URL_TEST", "postgres:///Forem_test") %>
+  database: <%= ENV.fetch("DATABASE_NAME_TEST", "Forem_test") %><%= ENV.fetch("TEST_ENV_NUMBER", "") %>
```

Comment:
no need to use `.fetch` for `TEST_ENV_NUMBER` if the default is an empty string, `nil` will be converted to empty string by `<%=`